src/main/java/io/gatling/mojo/EnterpriseStartMojo.java

```diff
@@ -143,19 +136,20 @@ private RunSummary startExistingSimulation(File file) throws MojoFailureExceptio
     }
   }
 
-  private RunSummary batchCreateAndStartSimulation(File file, UUID teamIdUuid, UUID packageIdUuid)
+  private RunSummary createAndStartSimulation(
+      EnterprisePlugin enterprisePlugin, File file, UUID teamIdUuid, UUID packageIdUuid)
       throws EnterprisePluginException, MojoFailureException {
-    getLog().info("Creating and starting simulation (batch mode)...");
-
-    final EnterprisePlugin enterprisePlugin = initEnterprisePlugin();
+    if (session.getRequest().isInteractiveMode()) {
+      getLog().info("Creating and starting simulation (batch mode)...");
```

Comment:
IMO, log is not needed (log that are not about configuration should be in commons).
But isInteractiveMode lead to batch mode log ?